File: src/main/java/com/p2p/lending/dao/UserauditorDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Userauditor;

public interface UserauditorDao {
	
	public  List<Userauditor> queryUserauditor(Map<String, Object> map);
	
	public int addUserauditor(Map<String, Object> map);

}

File: src/main/java/com/p2p/lending/repository/UserauditorDao.kt
package com.p2p.lending.repository

import com.p2p.lending.entity.Userauditor

import org.springframework.stereotype.Repository

@Repository
interface UserauditorDao {

    fun queryUserauditor(map: Map<String, Any?>): List<Userauditor>

    fun addUserauditor(map: Map<String, Any?>): Int

}
